homeassistant/components/volvooncall/config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Config Flow Handler for volvooncall."""
+import voluptuous as vol
+from volvooncall import Connection
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_REGION, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+
+from . import VolvoData
+from .const import CONF_MUTABLE, CONF_SCANDINAVIAN_MILES, DOMAIN
+from .errors import AuthenticationError, InvalidRegionError
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_REGION): cv.string,
+        vol.Optional(CONF_MUTABLE, default=True): cv.boolean,
+        vol.Optional(CONF_SCANDINAVIAN_MILES, default=False): cv.boolean,
+    },
+)
+
+
+class VolvoOnCallConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """VolvoOnCall config flow."""
+
+    VERSION = 1
+    data = None
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
+        """Handle user step."""
+        errors = {}
+        if user_input is not None:
+            try:
+                await self.is_valid(user_input)
+            except InvalidRegionError:
+                errors[CONF_REGION] = "invalid_region"
+            except AuthenticationError:
+                errors["base"] = "auth"
+            if not errors:
+                self.data = user_input
+                return self.async_create_entry(title="Volvo On Call", data=self.data)
```

Comment:
We don't need to store the user input. Maybe use the username as title instead? The name of the integration will show in the integration card anyway.
```suggestion
                return self.async_create_entry(title="Volvo On Call", data=user_input)
```

Reply:
Can we use the username to set a unique id of the config entry?

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids